openedx/core/lib/logsettings.py

```diff
@@ -126,6 +128,11 @@ def log_python_warnings():
     warnings.filterwarnings('ignore', 'Setting _field_data is deprecated')
     warnings.filterwarnings('ignore', 'Setting _field_data via the constructor is deprecated')
     warnings.filterwarnings('ignore', '.*unclosed.*', category=ResourceWarning)
+    warnings.filterwarnings('ignore', '.*You can remove default_app_config.*', category=RemovedInDjango41Warning)
+    warnings.filterwarnings('ignore', 'Instead access HTTPResponse.headers directly.*', category=DeprecationWarning, module='elasticsearch')
+    warnings.filterwarnings('ignore', 'Using or importing the ABCs from \'collections\' instead of from \'collections.abc\' is deprecated.*', category=DeprecationWarning, module="sass")
+    warnings.filterwarnings('ignore', 'Deprecated call to `pkg_resources.declare_namespace.*', category=DeprecationWarning)
+    warnings.filterwarnings('ignore', '.*pkg_resources is deprecated as an API.*', category=DeprecationWarning)
```

Comment:
We could also add `warnings.filterwarnings('ignore', "'etree' is deprecated. Use 'xml.etree.ElementTree' instead.", category=DeprecationWarning, module='wiki')`.